src/Shared/UnitTests/ObjectModelHelpers.cs

```diff
@@ -1936,5 +1939,126 @@ public void Dispose()
                 _buildManager.Dispose();
             }
         }
+
+        internal class LoggingFileSystem : IMSBuildFileSystem
+        {
+            private readonly IFileSystem _wrappingFileSystem;
+            private int _fileSystemCalls;
+
+            public int FileSystemCalls => _fileSystemCalls;
+
+            public ConcurrentDictionary<string, int> ExistenceChecks { get; } = new ConcurrentDictionary<string, int>();
```

Comment:
I didn't know this was a thing 😮

Reply:
Not to be confused with `=>`. Been there done that :)